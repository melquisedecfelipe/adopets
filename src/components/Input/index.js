import React from 'react';

import { Input } from 'antd';

import './styles.scss';

export default function InputComponent({ defaultValue, onBlur, placeholder, type }) {
  return (
    <Input defaultValue={defaultValue} onBlur={onBlur} placeholder={placeholder} type={type} />
  );
}
