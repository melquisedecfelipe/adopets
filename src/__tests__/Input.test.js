import React from 'react';

import { shallow } from 'enzyme';

import Input from '../components/Input';

describe('Input testing with Enzyme', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<Input />);

    expect(wrapper).toMatchSnapshot();
  });
});
