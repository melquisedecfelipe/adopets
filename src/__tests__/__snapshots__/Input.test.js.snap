// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input testing with Enzyme renders without crashing 1`] = `ShallowWrapper {}`;
